pyproject.toml

```diff
@@ -0,0 +1,20 @@
+[build-system]
+requires = ["setuptools >= 61.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+
+[project]
+name = "vmcu_package"
+version = "0.0.5"
+description = "A package to manage the resources of the STM32 simulator "
+readme = "README.md"
+requires-python = ">=3.8"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
```

Comment:
Should be only linux tho

Reply:
But the package can be use in any Operative system, is true that to work with the simulator of c++ you will need to be in linux. But due to the c++ simulator not the vmcu_package. So I din't think was necessary to restrict the package.